src/core/Table/index.ts

```diff
@@ -0,0 +1 @@
+export * from './BasicTable';
```

Comment:
If this has only one table component then we can just use as **./Table**

Reply:
Also, I guess you need to export globally the `Table` component at `src/core/index.ts`
something like  `export * from './Table';` in order to get the table exported for npm.

Reply:
@S-ayanide @arkajyotiMukherjee should we have this Table as a core or lab component?